### ISSUE TITLE
bundle install fails (actionpack-5.2.4.1 revealed dependencies)

### ISSUE THREAD

**Issue by Findus23** [tootsuite/mastodon#12993]
### Expected behaviour

After checking out v3.1.0rc1 `bundle install` should install the latest dependencies

### Actual behaviour

>Fetching actionpack 5.2.4.1
> Downloading actionpack-5.2.4.1 revealed dependencies not in the API or the lockfile (rack (~> 2.0, >= 2.0.8)).
> Either installing with `--full-index` or running `bundle update actionpack` should fix the problem.

### Steps to reproduce the problem

Check out `v3.1.0rc1` tag and run `bundle install`

### Specifications

Mastodon v3.1.0rc1
```
mastodon@vServer:~/live$ bundle --version
Bundler version 2.1.4
mastodon@vServer:~/live$ ruby --version
ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-linux]
```
----

Disclaimer: I know nearly nothing about ruby package management.

I found some hints on the internet that one could edit the lockfile and indeed this fixed the issue for me:

```diff
diff --git a/Gemfile.lock b/Gemfile.lock
index d78465c8f..5f7f72770 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -44,7 +44,7 @@ GEM
     actionpack (5.2.4.1)
       actionview (= 5.2.4.1)
       activesupport (= 5.2.4.1)
-      rack (~> 2.0, >= 2.1.2)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
```

Probably just updating the lockfile would also fix this.

**Comment by rodti** [comment]
Thanks for raising this, it was driving me mad!

**Comment by ThibG** [comment]
Fixed in 3.1.0rc2 that just released!

**Comment by rodti** [comment]
Woohoo! Thanks @ThibG
